api/users/models.py

```diff
@@ -121,6 +122,28 @@ def __str__(self):
     def subscribe_to_mailing_list(self):
         mailer_lite.subscribe(self)
 
+    @hook(AFTER_UPDATE, when="email", has_changed=True)
+    def send_warning_email(self):
+        message = inspect.cleandoc(
+            f"""
+            Hi there,
+
+            Your new email address is {self}.
+
+            If you haven't request this change please contact us at support@flagsmith.com
+
+            The Flagsmith Team
+            """
+        )
+
+        send_mail(
```

Comment:
We should delegate the sending of this email to the asynchronous processor. You can see an example of this being used [here](https://github.com/Flagsmith/flagsmith/blob/main/api/users/tasks.py#L9) but we can also go through this together next week.